### ISSUE TITLE
Create template for documentation

### ISSUE THREAD

**Issue by ChrsBaur** [at-gmbh/at-python-template#77]
It's important to create the documentation at the same time as you code your project, otherwise you won't ever do it or you'll die trying.

To create a template for documentation, MkDocs or Sphinx could be a solution.

 [MkDocs](https://www.mkdocs.org/) 
[Sphinx](https://www.sphinx-doc.org/en/master/)

**Comment by klamann** [comment]
yeah, we're doing that already with [personio-py](https://at-gmbh.github.io/personio-py/). I'm not sure if it's worth the effort, since we have all relevant information in the Readme and the [wiki](https://github.com/at-gmbh/at-python-template/wiki). What do you think would be the benefits?
